requirements.txt

```diff
@@ -0,0 +1,26 @@
+astroid==3.1.0
+axe-core-python==0.1.0
```

Comment:
axe should be removed and... we can merge

Reply:
done ;)